src/Network/Wai/SAML2/EntityDescriptor.hs

```diff
@@ -36,8 +36,8 @@ data IDPSSODescriptor
     = IDPSSODescriptor {
         -- | IdP Entity ID. 'Network.Wai.SAML2.Config.saml2ExpectedIssuer' should be compared against this identifier
         entityID :: Text
-        -- | The X.509 certificate for signed assertions
-    ,   x509Certificate :: X509.SignedExact X509.Certificate
+        -- | The X.509 certificate for signed assertions -- @since 0.7
```

Comment:
Also, the `@since` annotation doesn't work at the end of that line -- it has to be on a separate line I think (and possibly separated by an empty comment line)

Reply:
Good catch, fixed